jobs/jobs.go

```diff
@@ -98,8 +98,16 @@ func FromConfigs(cfgs []*Config) []*Job {
 	return jobs
 }
 
-// SendHeartbeat sends a heartbeat for this Job's service
-func (job *Job) SendHeartbeat() {
+// registerUnhealthyService registers this Job's service with status set to
+// critical. Will only register the service if it does not already exist.
+func (job *Job) registerUnhealthyService() {
+	if job.Service != nil {
+		job.Service.RegisterUnhealthy()
+	}
+}
+
+// sendHeartbeat sends a heartbeat for this Job's service
+func (job *Job) sendHeartbeat() {
```

Comment:
I figured since the tests pass this isn't a problem but what were your thoughts on changing this from an exported method? `Job.SendHeartbeat` => `Job.sendHeartbeat`?

Reply:
I had a hard time seeing how that method will be used outside of the job itself, but it's a small change so I can change it back if you'd like.